src/js/auth.js

```diff
@@ -80,21 +84,23 @@ logout.addEventListener('click', (e) => {
 
 
 
+
+
+
 // login
-const loginForm = document.querySelector('#login-form');
-loginForm.addEventListener('submit', (e) => {
+const LogInBtn = document.querySelector('#log-in-btn');
```

Comment:
Please keep formatting consistent, all variables should start with lowercase letter